src/language/branches.md

```diff
@@ -179,12 +184,14 @@ token = switch (charCode) {
 Exhaustiveness checking is a feature that reports a compile-time
 error if it's possible for a value to enter a switch but not match any of the cases.
 
+<?code-excerpt "language/lib/control_flow/branches.dart (exh-bool)"?>
 ```dart
 bool? b = false;
-// Non-exhaustive switch on bool?, missing case to match null possiblity:
```

Comment:
Losing this comment makes it not clear that this is intended to show a non-exhaustive example, which seems to be the intention.